doc/concepts/configuration/configuration_etcd.rst

```diff
@@ -6,92 +6,150 @@ Storing configuration in etcd
 
-        $ tt cluster publish "http://localhost:2379/tt" remote_config.yaml
+-   :ref:`config.etcd.username <config_etcd_username>` and :ref:`config.etcd.password <config_etcd_password>` specify credentials used for authentication.
+-   :ref:`config.etcd.ssl.ca_file <config_etcd_ssl_ca_file>` specifies a path to a trusted certificate authorities (CA) file.
+-   :ref:`config.etcd.http.request.timeout <config_etcd_http_request_timeout>` configures a timeout for connecting to an etcd server.
 
-    Searches keys by the following path: ``/prefix/config/*``.
-    See https://github.com/tarantool/doc/issues/3725
+You can find all the available configuration options in the :ref:`etcd <configuration_reference_config_etcd>` section.
 
-    Manual:
 
-    .. code-block:: yaml
 
-        config:
-          reload: 'manual'
+.. _etcd_publishing_configuration:
 
-    Reload config (on all instances):
+Publishing a cluster's configuration to etcd
+--------------------------------------------
 
-    .. code-block:: lua
+.. _etcd_publishing_configuration_tt:
 
-        require('config'):reload()
+Publishing configuration using the tt utility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    Authentication:
+The tt utility provides the :ref:`tt cluster <tt-cluster>` command for managing a cluster's configuration.
+The ``tt cluster publish`` command can be used to publish a cluster's configuration to etcd.
 
-    .. code-block:: console
+The example below shows how a :ref:`layout <admin-start_stop_instance-multi-instance-layout>` of the application called ``app`` might look:
 
-        $ etcdctl --user root --password foobar role grant-permission tt readwrite /tt/config/all
-        $ etcdctl --user root --password foobar role grant-permission tt --prefix=true readwrite /tt/
+.. code-block:: none
 
-        $ etcdctl --user root --password foobar user grant-role testuser tt
+    instances.enabled
+    └── app
+        ├── config.yaml
+        ├── cluster.yaml
+        └── instances.yml
 
+*   ``config.yaml`` contains a :ref:`local configuration <etcd_local_configuration>` used to connect to etcd.
+*   ``cluster.yaml`` contains a cluster's configuration to be published.
+*   ``instances.yml`` specifies :ref:`instances <admin-start_stop_instance-multi-instance>` to run in the current environment. ``tt cluster publish`` ignores the configured instances.
 
-    .. code-block:: yaml
+To publish a cluster's configuration (``cluster.yaml``) to an etcd server, execute ``tt cluster publish`` as follows:
 
-        config:
-          etcd:
-            http:
-              request:
-                timeout: 3
-            prefix: /tt
-            endpoints:
-            - http://localhost:2379
-            username: testuser
-            password: foobar
+.. code-block:: console
 
+    $ tt cluster publish "http://localhost:2379/example" instances.enabled/app/cluster.yaml
```

Comment:
Hm. I need a local config to place etcd to config to etcd -- said the article somewhere above. But here we need to specify the etcd URI and prefix. So, a local config doesn't involved here, right?

Reply:
Good question.. I've checked `tt cluster publish` behavior one more time:
- `tt cluster publish "http://localhost:2379/example" instances.enabled/app/cluster.yaml` doesn't require `config.yaml` at all and publishes `cluster.yaml` using the URI.
- `tt cluster publish app instances.enabled/app/remote_config.yaml` does require `config.yaml` but works a bit strange to me:
    1) doesn't update a corresponding key on a server using etcd connection settings from `config.yaml`.
    2) overwrites `config.yaml` with the `cluster.yaml` contents.